cypress/support/commands.ts

```diff
@@ -592,13 +594,20 @@ export const createToken = (
   })
 }
 
-export const setupUser = (): Cypress.Chainable<any> => {
+export const setupUser = (useIox: boolean = false): Cypress.Chainable<any> => {
+  useIox = Cypress.env('useIox') || useIox
   const defaultUser = Cypress.env('defaultUser')
-  const userParam = defaultUser ? `?user=${defaultUser}` : ''
+  const params = new URLSearchParams()
+  if (defaultUser) {
+    params.set('user', defaultUser)
+  }
+  if (useIox) {
+    params.set('orgSuffix', 'ioxlighthouse')
+  }
   return cy
     .request({
       method: 'GET',
-      url: `/debug/provision${userParam}`,
+      url: `/debug/provision${params.toString()}`,
```

Comment:
`params.toString()` doesn't include the `?` identifier, so you'll need to add it here (but it should be fine if `params.toString()` is an empty string, since an empty query string with `?` shouldn't hurt anything)

```ts
`/debug/provision?${params.toString()}`
```

Reply:
ah nice catch